changelog/example.json

```
@@ -0,0 +1,6 @@
+[
+  {
+    "type": "Features",
+    "title": "Description of changes"
+  }
```

```suggestion
  },
  {
    "type": "Bug fixes and improvements",
    "title": "Description of changes"
  }
```

thinking it's good to have the example include `type` possibilities